AwsEncryptionSDK/dafny/AwsEncryptionSdk/Model/AwsCryptographyEncryptionSdkTypes.dfy

```diff
@@ -187,13 +187,20 @@ abstract module AbstractAwsCryptographyEncryptionSdkService
   import Operations : AbstractAwsCryptographyEncryptionSdkOperations
   function method DefaultAwsEncryptionSdkConfig(): AwsEncryptionSdkConfig
   method ESDK(config: AwsEncryptionSdkConfig := DefaultAwsEncryptionSdkConfig())
-    returns (res: Result<ESDKClient, Error>)
+    returns (res: Result<IAwsEncryptionSdkClient, Error>)
```

Comment:
Hum, I must have missed this but why change to the interface? (I know this is `smithy-dafny` generated)

Reply:
See above.